ansible-contrib/roles/master/tasks/iptables.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Get iptables rules
+  command: iptables -L --wait
+  register: iptablesrules
+  always_run: yes
+
+- name: Open apiserver port with iptables
+  command: /sbin/iptables -I INPUT 1 -p tcp --dport {{ kube_master_api_port }} -j ACCEPT -m comment --comment "kube-apiserver"
+  when: "'kube-apiserver' not in iptablesrules.stdout"
+  notify:
+    - restart iptables
+
+- name: Save iptables rules
+  command: service iptables save
```

Comment:
Probably a script in `/etc/network/if-pre-up.d/iptables` is more reliable & more portable?